README.md

```diff
@@ -409,24 +410,24 @@ Message.event "Hi {user}!"
 |> Message.setFieldValue "user" "haf"
 ```
 
-And then targets that support templating will output them 'filled out'.
+This enables targets that support templating to output them 'filled out'.
 
 Message Templates are a superset of standard .NET format strings, so any format
 string acceptable to string.Format() will also be correctly processed by
 FsMessageTemplates.
 
- * Property names are written between { and } brackets
- * Brackets can be escaped by doubling them, e.g. {{ will be rendered as {
- * Formats that use numeric property names, like {0} and {1} exclusively, will
-   be matched with the Format method's parameters by treating the property names
-   as indexes; this is identical to string.Format()'s behaviour
+ * Property names are written between `{` and `}` braces
```

Comment:
Interesting. Made me [read through this](https://english.stackexchange.com/questions/3379/bracket-vs-brace).